tests/support/facilityId.ts

```diff
@@ -0,0 +1,43 @@
+import { execSync } from "child_process";
+import fs from "fs";
+import path from "path";
+
+const META_PATH = path.resolve("tests/.auth/facilityMeta.json");
+let cachedId: string | null = null;
+
+/**
+ * Returns the facilityId saved during setup.
+ * Auto-runs the setup if the meta file is missing or invalid.
+ */
+export function getFacilityId(): string {
+  if (cachedId) return cachedId;
+
+  if (!fs.existsSync(META_PATH)) {
+    console.warn("⚠️ Facility meta missing — running facility setup...");
+    try {
+      execSync(
+        "npx playwright test --project=setup tests/setup/facility.setup.ts",
+        {
+          stdio: "inherit",
+          cwd: process.cwd(),
+        },
+      );
+    } catch (error) {
+      throw new Error(
+        `Failed to run facility setup: ${error instanceof Error ? error.message : error}`,
+      );
+    }
+  }
+
+  const raw = fs.readFileSync(META_PATH, "utf8");
```

Comment:
For future, we should either delete this file (regardless of tests fail or pass) or validate the Id, will be helpful for local testing (if/when we rerun fixtures script).